tests/testthat/test-summary.R

```diff
@@ -16,7 +16,7 @@ withr::with_options(list(rbabylon.bbi_exe_path = '/data/apps/bbi',
   MOD2_PATH <- file.path(MODEL_DIR, "2")
 
   # references
-  SUMMARY_REF_FILE <- "data/acop_summary_obj_ref_200520.rds"
+  SUMMARY_REF_FILE <- "data/acop_summary_obj_ref_200616.rds"
```

Comment:
How is this file created?

Reply:
`sum1 <- MOD1 %>% model_summary()`

Whenever one of these tests breaks I run this and compare `sum1` to `readRDS(SUMMARY_REF_FILE)` manually. Usually it's some little structural thing that's changed and is causing it to break. If they look good, I resave `sum1` out to a new reference.

Do you think we should document that somewhere?

Reply:
Yes, I think that we should document the creation of any artifacts used in testing, but we can take that as a separate issue after we merge this.